framework/src/modules/governance/stores/index.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { BaseStore } from '../../base_store';
+
+export interface IndexStoreData {
```

Comment:
index has special meaning so maybe we can change the name to `ProposalIndex`?

Reply:
True. I also didn't like calling it `index`.